src/lib/y2partitioner/actions/resize_blk_device.rb

```diff
@@ -110,6 +114,18 @@ def confirm_empty
         Yast2::Popup.show(message, headline: :warning, buttons: :yes_no) == :yes
       end
 
+      # Resizing a thin snapshot does not make too much sense. The user has to confirm a warning
+      # before continue.
+      #
+      # @return [Boolean] true if device is not a thin snapshot or user decided go ahead
+      def confirm_thin_snapshot_resize
+        return true unless @device.is?(:lvm_thin_snapshot)
+
+        message = _("Selected device is a Thin Snapshot LV. Do you really want to resize it?")
```

Comment:
Again, I would avoid the term "LV" here.

Reply:
Let's use "...is an LVM Thin Snapshot", do you agree?